src/test/java/com/revature/services/JMeterResponseCollectorTest.java

```diff
@@ -0,0 +1,213 @@
+package com.revature.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+
+import org.apache.jmeter.engine.StandardJMeterEngine;
+import org.apache.jmeter.reporters.Summariser;
+import org.apache.jmeter.samplers.SampleEvent;
+import org.apache.jmeter.samplers.SampleResult;
+import org.apache.jmeter.util.JMeterUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.revature.docutest.TestUtil;
+import com.revature.responsecollector.JMeterResponseCollector;
+import com.revature.templates.LoadTestConfig;
+
+class JMeterResponseCollectorTest {
+    private JMeterResponseCollector jmrc;
+    private StandardJMeterEngine engine;
+    private static final String JMeterPropPath = "src/test/resources/test.properties";
+    private SampleEvent event1;
+    private SampleEvent event2;
+    private SampleEvent event3;
+    private SampleEvent event4;
+    private SampleEvent event5;
+    private SampleResult event1Result;
```

Comment:
Maybe these could be arrays?

Reply:
We decided to use Collections, so we can arbitrarily add/remove events.